libs/log/cmtfmt_logger.go

```diff
@@ -92,7 +92,8 @@ func (l tmfmtLogger) Log(keyvals ...any) error {
 
 		// Realize stringers
 		if s, ok := keyvals[i+1].(fmt.Stringer); ok {
-			keyvals[i+1] = s.String()
```

Comment:
Please help me understand one thing: if `s` is `nil`, how come it passes the check for `fmt.Stringer` (`ok`) and then panics?

Reply:
`keyvals[i+1]` is nil  but `s` isn't a `nil`. you can run the above example to get the panic.